src/cow-react/modules/wallet/index.ts

```diff
@@ -1,9 +1,16 @@
 export * from './api/types'
-export * from './api/hooks/useWalletDetails'
 
-export * from './web3-react/hooks/useWalletInfo'
+// Hooks
+export * from './api/hooks'
 export * from './web3-react/hooks/useDisconnectWallet'
+export * from './web3-react/hooks/useWalletMetadata'
+export * from './web3-react/hooks/useEagerlyConnect'
+export * from './web3-react/hooks/useOrderedConnections'
 
+// Updater
+export * from './web3-react/updater'
+
+// Components
 export * from './web3-react/containers/Web3Status'
```

Comment:
This `index` is serving the porpouse of a facade of the module.

Everythin that is exposed here has a simple interface that don't depend in web3-react. 

This way, we can import from `@cow/modules/wallet` all relevant things, like the updater, hooks, or types